server/tests/clients/test_google.py

```diff
@@ -124,3 +128,101 @@ def test_get_user_info_happy() -> None:
     assert userInfo["email"] == myEmail
     assert userInfo["id"] == myId
     assert userInfo["picture"] == myPicture
+
+
+def test_get_google_person_list_happy() -> None:
```

Comment:
Please add a happy path that has 2+ persons returned by the api.

Reply:
added